docs/source/conf.py

```diff
@@ -21,13 +21,13 @@
 # -- Project information -----------------------------------------------------
 
 project = "Font Bakery"
-copyright = "2018, Google Inc. CC BY-SA 4.0 License"
-author = "Google Inc."
+copyright = "Copyright 2013 The Font Bakery Authors"
+author = "The Font Bakery Authors"
 
 # The short X.Y version
-version = ""
+version = "0.6"
 # The full version, including alpha/beta/rc tags
-release = ""
+release = "0.6.0"
```

Comment:
See https://github.com/googlefonts/fontbakery/pull/2106#discussion_r231252464

Reply:
Regardless of this, I will merge in order to have the other contribs from Dave on master. Fee free to send further PRs if you believe there are better ways to handle this.